src/components/Direction.js

```diff
@@ -30,27 +31,273 @@ export default class Direction extends Component {
 		}
 		
 		return (
-			<ScrollView>
-				<Text style={styles.heading}>Getting from {this.state.data.origin} to {this.state.data.destination}</Text>			
-				<FlatList
-					scrollEnabled={this.state.scrollable}
-					data={this.state.data.images}
-					renderItem={({item}) => 
-						<View style={styles.container}>
-							<Image
-								style={styles.image}
-								source={item.src}
-							/>
-							<Text style={styles.item}>{item.desc}</Text>
-						</View>
-					}
-				/>
-				<FlatList
-					scrollEnabled={this.state.scrollable}
-					data={this.state.data.instructions}
-					renderItem={({item, index}) => <Text style={styles.item}>{index+1}. {item}</Text>}
-				/>
-			</ScrollView>
+			<View style={styles.container}>
```

Comment:
Please revert the changes to `Direction.js`. It's outside the scope for this PR.